test/parallel/test-crypto-hmac.js

```diff
@@ -221,12 +221,12 @@ var rfc4231 = [
   }
 ];
 
-for (var i = 0, l = rfc4231.length; i < l; i++) {
-  for (var hash in rfc4231[i]['hmac']) {
-    var str = crypto.createHmac(hash, rfc4231[i].key);
+for (let i = 0, l = rfc4231.length; i < l; i++) {
+  for (const hash in rfc4231[i]['hmac']) {
```

Comment:
Yeah, it's a bit surprising at first, but the value is scoped to the subsequent block.

This:

```
'use strict';

for (const foo of ['a', 'b', 'c']) {
  console.log(foo, typeof foo);
}

console.log(typeof foo);
```

...yields this output:

```
a string
b string
c string
undefined
```